defaults/sites_to_mask.txt

```diff
@@ -0,0 +1,12 @@
+1820
```

Comment:
Do we have an external reference for this list of sites that we could mention as a comment in this file or in the commit message? Or are these sites that we found as problematic? Either way, if we support comments in this file, that could help future us.

Reply:
I don't think we support comments here, though I agree it would be nice to give masking reasons and at least the corresponding amino acid changes. This is the augur function that parses it:
https://github.com/nextstrain/augur/blob/5ad157485015623883c6b637d247459f906b63cb/augur/utils.py#L700

Reply:
The sites mutated convergently in many important clades and include things like S:501Y.